components/tooltip.scss

```diff
@@ -1,98 +1,121 @@
-$tooltip-tip-size: 10px;
-$tooltip-padding: 0.5rem 1rem;
-$tooltip-width: 150px;
-$tooltip-menu-side-padding: 0.75rem;
-$tooltip-menu-padding: 0.5rem 1rem;
-$tooltip-tip-right-offset: 1rem;
-$tooltip-tip-left-offset: 1rem;
-$tooltip-border-radius: $border-radius;
-
 .hdd-tooltip {
-  // Variable containing '&' to jump to current parent inside a child class.
   $this: &;
-
   position: relative;
+  display: inline-block;
 
-  &:hover:not(.no-hover) {
+  &:hover {
+    #{$this}_content {
+      @extend .hdd-tooltip_content, .is-active;
+    }
+  }
+  &:focus {
     #{$this}_content {
       @extend .hdd-tooltip_content, .is-active;
     }
   }
 
   &_content {
-    //.is-p is font-properties, defined in typography.
-    @extend .is-p;
-
-    $content-color: var(--hdd-color-grey-dark);
-    $content-text-color: var(--hdd-color-white);
-
-    display: table;
+    @extend .is-p-small;
     position: absolute;
-    z-index: -1;
-
-    top: calc(100% + #{$tooltip-tip-size});
-    right: 0;
-    left: 50%;
-
-    box-sizing: border-box;
-    min-width: $tooltip-width;
-
-    margin: auto;
-    padding: $tooltip-padding;
-
-    transform: translateX(-50%);
-    transition: opacity $medium-transition $transition-cubic, z-index $medium-transition step-end;
-    border-radius: $tooltip-border-radius;
-
     opacity: 0;
-    background: $content-color;
-    color: $content-text-color;
-    text-align: left;
-
-    &::before {
-      content: '';
-
-      display: block;
-      position: absolute;
-
-      top: -$tooltip-tip-size;
-      right: 0;
-      left: 0;
-
-      width: 0;
-      height: 0;
-
-      margin: auto;
-
-      border-right: $tooltip-tip-size solid transparent;
-      border-bottom: $tooltip-tip-size solid $content-color;
-      border-left: $tooltip-tip-size solid transparent;
+    z-index: -1;
+    min-width: 90px;
+    max-width: 70ch;
+    background-color: var(--hdd-color-grey-dark);
+    color: var(--hdd-color-contrast-font-color);
+    padding: 10px var(--hdd-spacing-2);
+    border-radius: $border-radius;
+    box-shadow: $shadow-strong;
+    transition: 150ms;
+    transform: scale(0.5);
+
+    // Defaults to is-bottom tooltip if no direction class is applied.
+    &:not(.is-right):not(.is-left):not(.is-top) {
+      @extend .is-bottom;
     }
 
-    &.is-active {
-      z-index: 90;
-      transition: opacity $medium-transition $transition-cubic, z-index $medium-transition step-start;
-      opacity: 1;
+    &.is-right {
+      top: -10px;
+      left: 110%;
+      margin-left: 0;
+
+      &::after {
+        content: "";
+        position: absolute;
+        top: 50%;
+        right: 100%;
+        margin-top: -5px;
+        border: 5px solid;
+        border-color: transparent var(--hdd-color-grey-dark) transparent transparent;
```

Comment:
Antar dette er for å tegne pilene. Kan de gjøres til en mixin, eller bruke top-arrow?